### DISCUSSION TITLE
When is the next version of Flowbite scheduled for release?

### DISCUSSION THREAD

**Discussion by fiqryq** [themesberg/flowbite-react#1485] [General]
In the last version, there was a bug with the datepicker component, which caused double months to appear on the initial render. I’d like to know when the next update will be available.

**Comment by fiqryq** [comment]
cc : @adeherysh

**Comment by fiqryq** [comment]
Hello i want to follow up this question. 🆙

**Reply by fiqryq** [reply]
cc @flowbite

**Reply by SutuSebastian** [reply]
Today
